File: C11/백준/Bronze/10869. 사칙연산/사칙연산.c
#include <stdio.h>

int main(void){
	
	int A = 0;
	int B = 0;
	scanf("%d %d", &A, &B);
	
	int sum = A + B;
    int min = A - B;
    int mul = A * B;
    int div = A / B;
    int mod = A % B;
	
	printf("%d\n", sum);
	printf("%d\n", min);
	printf("%d\n", mul);
	printf("%d\n", div);
	printf("%d", mod);
	
	return 0;
} 

File: Java/백준/Bronze/10869. 사칙연산/사칙연산.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {

        Scanner s = new Scanner(System.in);

        int A = s.nextInt();
        int B = s.nextInt();

        int sum = A + B;
        int min = A - B;
        int mul = A * B;
        int div = A / B;
        int mod = A % B;
        
        System.out.println(sum);
        System.out.println(min);
        System.out.println(mul);
        System.out.println(div);
        System.out.println(mod);

    }
}